### PR TITLE
Some improvements of scheduler

### PR DESCRIPTION
**Pull request by shinytang6**
some minor improvements while scanning the scheduler code
Signed-off-by: tangliang <1074461480@qq.com>

### CODE REVIEW

**Comment by Thor-wl** [comment]
@shinytang6 please resolve the conflictions.

File: pkg/scheduler/api/node_info.go [hunk 1]
```diff
@@ -283,7 +283,7 @@ func (ni *NodeInfo) RemoveTask(ti *TaskInfo) error {
 			ni.Releasing.Sub(task.Resreq)
 			ni.Idle.Add(task.Resreq)
 			ni.Used.Sub(task.Resreq)
-			ni.AddGPUResource(ti.Pod)
+			ni.SubGPUResource(ti.Pod)
```
Comment (william-wang) [inline pkg/scheduler/api/node_info.go:286]: Why decreace GPU resource?

Comment (shinytang6) [reply pkg/scheduler/api/node_info.go:286]: Because it's removing task from node, how could we add resource here?

**Pull request merged**